docker/backend-dev/conf/Config2.pm

```diff
@@ -39,23 +39,30 @@ BEGIN
 		$memd_servers
 		$facebook_app_id
 	    $facebook_app_secret
+		$google_cloud_vision_api_key
+		$crowdin_project_identifier
+		$crowdin_project_key
+		$csrf_secret
 		$robotoff_url
 		%server_options
 	);
 	%EXPORT_TAGS = (all => [@EXPORT_OK]);
 }
 use vars @EXPORT_OK ; # no 'my' keyword for these
+use strict;
+use utf8;
 
 # server constants
-$server_domain = "productopener.localhost";
+$server_domain = $ENV{PRODUCT_OPENER_DOMAIN};
```

Comment:
Shouldn't we keep the || "productopener.localhost"; part ?

e.g. for development, I run start_dev.sh without any environment variables defined. It would be nice to keep a default URL.

Reply:
It will be the default already in `docker/.env` so there is no need to duplicate.

Reply:
`.env` is loaded when `docker-compose` runs :) so in the local environment, the repo's `.env` will be loaded